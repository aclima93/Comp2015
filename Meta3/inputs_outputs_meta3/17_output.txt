Line 24, col 9: Type identifier expected
