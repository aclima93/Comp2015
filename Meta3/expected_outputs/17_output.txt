Line 25, col 7: Symbol IsOperator not defined
